board/ti/am62x-sk/rootfs-overlay/etc/init.d/S99weston

```diff
@@ -16,13 +16,25 @@ start() {
     # Start Weston
     printf "Starting weston: "
     /usr/bin/weston > /var/log/weston.log 2>&1 &
-    [ $? = 0 ] && echo "OK" || echo "FAIL"
+    # Wait for a short time and check if Weston is running
+    sleep 2
```

Comment:
I doubt if this would be ideal.
sleep is prone to failures.

Why didn't the existing method work?
Also, is there any better way to handle it instead of delaying the boot by 2 secs?